cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -84,33 +100,109 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
         p("Nothing is returned from this query, except the count of system database changes made.")
         resultTable()
       }
-      p("The privileges granted to the `regularUser` role will appear on the list provided by `SHOW ROLE $role PRIVILEGES`.")
-      query("SHOW ROLE regularUser PRIVILEGES", assertPrivilegeShown(Seq(Map("grant" -> "GRANTED", "segment" -> "NODE(Post)", "role" -> "regularUser")))) {
-        resultTable()
-      }
     }
+
     section("The READ privilege", "administration-security-subgraph-read") {
-      p("Users can be granted the right to do property reads on nodes and relationships using the `GRANT READ` privilege")
+      p(
+        """Users can be granted the right to do property reads on nodes and relationships using the `GRANT READ` privilege.
+          |It is very important to note that users can only read properties on entities that they is allowed to find in the first place.""".stripMargin)
       p("include::grant-read-syntax.asciidoc[]")
 
+      p(
+        """For example, we can allow the user `jake`, who has role 'regularUser' to read all properties on nodes with the label `Post`.
+          |The `*` implies that the ability to read all properties also extends to properties that might be added in the future.""".stripMargin)
+      query("GRANT READ { * } ON GRAPH neo4j NODES Post TO regularUser", ResultAssertions((r) => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        p("Nothing is returned from this query, except the count of system database changes made.")
+        resultTable()
+      }
+
       p("The `READ` privilege can also be denied.")
       p("include::deny-read-syntax.asciidoc[]")
 
+      p("Although we just granted the user 'jake' the right to read all properties, we may want to hide the `secret` property. The following example shows how to do that.")
+      query("DENY READ { secret } ON GRAPH neo4j NODES Post TO regularUser", ResultAssertions((r) => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        p("Nothing is returned from this query, except the count of system database changes made.")
+        resultTable()
+      }
     }
     section("The MATCH privilege", "administration-security-subgraph-match") {
       p("As a shorthand for `TRAVERSE` and `READ`, users can be granted the right to find and do property reads on nodes and relationships using the `GRANT MATCH` privilege. ")
+      p("Please note that `REVOKE MATCH` is not allowed.")
```

Comment:
Would like to have this as a note instead of a paragraph (and maybe last in the section?)